projects/hipdnn/frontend/include/hipdnn_frontend/attributes/Attributes.hpp

```diff
@@ -84,4 +82,3 @@ class Attributes
     }
 };
 }
```

Comment:
```suggestion
} // namespace hipdnn_frontend::graph
```
Just pointing these out, but again nit.